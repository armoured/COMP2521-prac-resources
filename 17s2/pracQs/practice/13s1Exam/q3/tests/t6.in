8
0-1
0-2
1-3
2-3
4-5
4-6
5-7
5-6
5-7
6-7
