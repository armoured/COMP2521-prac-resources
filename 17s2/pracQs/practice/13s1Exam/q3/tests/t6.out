Graph:
# vertices = 8, # edges = 9

0 is connected to 1 2
1 is connected to 0 3
2 is connected to 0 3
3 is connected to 1 2
4 is connected to 5 6
5 is connected to 4 6 7
6 is connected to 4 5 7
7 is connected to 5 6

# connected components = 2
